src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -333,6 +333,11 @@ public string PythonTemplateFilePath
         /// </summary>
         public bool ShowTabsAndSpacesInScriptEditor { get; set; }
 
+        /// <summary>
+        /// Engine used by default for new Python script and string nodes. If defined this takes precedence over any system settings.
```

Comment:
do you mean if not null? Does empty string qualify as defined?

Reply:
I'm pretty sure I'm using empty string for not defined.

Reply:
can you document that in this comment?